CONTRIBUTING.md

```diff
@@ -122,6 +121,7 @@ snforge_std.workspace = true
 
 [dev-dependencies]
 assert_macros.workspace = true
```

Comment:
I'm not sure why many packages don't have `assert_macros` declared in their tomls, and yet they work just fine.
AFAIK this package should be [declared explicitly from 2.8.1 at least](https://github.com/software-mansion/scarb/releases/tag/v2.8.1)

Reply:
`assert_macros` is included in `cairo_test`
However `cairo_test` is not compatible with starknet foundry, so for listing using snforge we have to explicitly import it.
See [Adding a new Cairo program](https://github.com/NethermindEth/StarknetByExample/blob/main/CONTRIBUTING.md#adding-a-new-cairo-program)